content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Some high level non-blocking comments here. I think it might be better to restructure this into a rough outline like this:

**Intro**
Today we're releasing ingress2gateway, a tool that can help you migrate from Ingress to Gateway API. With Gateway API just weeks away from graduating to GA, it's time to consider upgrading if you haven't already...

**Why Upgrade?**
* New Features
* Role-Oriented Design
* Supported on 5 latest Kubernetes versions
* Already over 20 implementations of the API
* Can use the same APIs to configure mesh routing

**How to upgrade**
* Gateway API Getting Started guide for CRD + Controller installation
* Ingress2Gateway
* Run both in parallels, cutover DNS after manually verifying GW API routing

**Further Reading**


I think you already have most of the content for this, but IMO restructuring this post to follow this general layout would help it flow a bit better.

Reply:
> Supported on 5 latest Kubernetes versions

since what version?

Reply:
changed it to be closer to the suggested flow - PTAL